### ISSUE TITLE
translation guidelines

### ISSUE THREAD

**Issue by aspina7** [appliedepi/epiRhandbook_eng#199]
rOpenSci have produced guidelines around how they to [localisation and translation](https://translationguide.ropensci.org/) might be worth taking a look and producing similar documentation for this our new infrastructure

**Comment by aspina7** [comment]
As can see in the readme of this [package](https://github.com/ropensci/datefixR?tab=readme-ov-file) they also create a tracker of completion using [gitlocalize](https://gitlocalize.com/#demo) - which may be worth looking at

**Comment by aspina7** [comment]
also crosslinking to this https://github.com/appliedepi/epiRhandbook_eng/issues/124 on the [{i18n}](https://github.com/rich-iannone/i18n) package

**Comment by aspina7** [comment]
and these https://github.com/R4EPI/sitrep/issues/314 on https://github.com/appliedepi/epiRhandbook_eng/issues/129 cleaning up how we store and translate datasets

**Comment by aspina7** [comment]
as well as considering how we should do this for package websites https://github.com/R4EPI/sitrep/issues/312

**Comment by ntluong95** [comment]
@aspina7 @nsbatra I just had a quick look on links you sent, I think the guideline document in the first comment recommending to use {babeldown} and {babelquarto} package, things we are using. The [gitlocalize](https://gitlocalize.com/#demo) looks interesting and can be investigated later

**Comment by aspina7** [comment]
yes @ntluong95 - I was just saying that we should plan to have one document on github (e.g. a wiki or a readme) or website. 
In this document we explain the whole workflow of translations for websites(books / case studies), courses/tutorials and datasets. 
This would be done in the same style as ropensci document linked above.

**Comment by ntluong95** [comment]
I see your point @aspina7. Could be a wiki or readme as the easiest option. Will keep it open to work in the near future
